app/frontend/src/components/AnalysisPanel/AnalysisPanel.tsx

```diff
@@ -1,5 +1,4 @@
 import { Stack, Pivot, PivotItem } from "@fluentui/react";
-import SyntaxHighlighter from "react-syntax-highlighter";
```

Comment:
Surprised we ever needed this

Reply:
That's used in the ThoughtProcess component, which moved into its own file. It makes our JSON slightly prettier than normal, in theory.